examples/stress_tests/many_animated_sprites.rs

```diff
@@ -26,7 +26,7 @@ fn main() {
             DefaultPlugins.set(WindowPlugin {
                 primary_window: Some(Window {
                     present_mode: PresentMode::AutoNoVsync,
-                    resolution: WindowResolution::new(1920.0, 1080.0)
+                    resolution: WindowResolution::new(LogicalSize::new(1920.0, 1080.0))
```

Comment:
I would go for `WindowResolution::from((1920.0, 1080.0))`.

This touches all of the examples that specify `WindowResolution`, would like to get someone else's opinion.

Reply:
I also agree on something like this, as I did feel that having to write `LogicalSize::new()` was a little verbose. I like `From` with the f32 tuple, but would reverting the change to `WindowResolution::new()` back to taking two f32, instead of `LogicalSize` be more preferable?

Reply:
I think the "main" constructor should be the "most correct". If needed, we can do a `from_xy` method.